calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventEntityProvider.java

```diff
@@ -46,19 +44,8 @@
 		ActionsExecutable, Outputable, Sampleable {
 
 	String ENTITY_PREFIX = "calendar";
-	Properties config = new Properties();
-	Map<String, String> eventImageMap = Collections.emptyMap();
-	private static Log log = LogFactory.getLog(CalendarEventEntityProvider.class);
-
-	public void init(){
-		try {
-			config.load(getClass().getResourceAsStream("/org/sakaiproject/calendar/tool/calendar.config"));
-		} catch (IOException e) {
-			//config file is missing
-			log.warn("Calendar.config file is missing for CalendarEventEntityProvider.class : " + e.getMessage());
-		}
-	}
-
+	Map<String, String> eventIconMap = Collections.emptyMap();
```

Comment:
I think this field can be made a variable inside the `getCalendarEventsForSite()` and `getMergedCalendarEventsForSite()` methods.